### ISSUE TITLE
Bug in the dataset_loader.py file

### ISSUE THREAD

**Issue by dearhoper** [isseu/emotion-recognition-neural-networks#14]
Hi @isseu ,
There are two mistakes in the dataset_loader.py file. I wonder if you need to revise them.
See [Line 17](https://github.com/isseu/emotion-recognition-neural-networks/blob/master/dataset_loader.py#L17) and [Line 19](https://github.com/isseu/emotion-recognition-neural-networks/blob/master/dataset_loader.py#L19).

**Comment by jorgimello** [comment]
Yeah, I found it too. 
After this step my training set and validation set had the same length. In the cvs conversion everything was alright.
